aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -117,36 +112,26 @@ module aptos_framework::account {
     native fun create_address(bytes: vector<u8>): address;
     native fun create_signer(addr: address): signer;
 
+    /// Only called during genesis to initialize system resources for this module.
     public(friend) fun initialize(
-        account: &signer,
-        module_addr: address,
-        module_name: vector<u8>,
+        aptos_framework: &signer,
         script_prologue_name: vector<u8>,
         module_prologue_name: vector<u8>,
-        writeset_prologue_name: vector<u8>,
         multi_agent_prologue_name: vector<u8>,
         user_epilogue_name: vector<u8>,
-        writeset_epilogue_name: vector<u8>,
     ) {
-        system_addresses::assert_aptos_framework(account);
+        system_addresses::assert_aptos_framework(aptos_framework);
 
-        move_to(account, ChainSpecificAccountInfo {
-            module_addr,
-            module_name,
+        move_to(aptos_framework, ChainSpecificAccountInfo {
+            module_addr: @aptos_framework,
+            module_name: b"account",
             script_prologue_name,
             module_prologue_name,
-            writeset_prologue_name,
             multi_agent_prologue_name,
             user_epilogue_name,
-            writeset_epilogue_name,
         });
-    }
 
-    // This should only be called during genesis.
-    public(friend) fun create_address_map(aptos_framework_account: &signer) {
```

Comment:
what was this?

Reply:
Unnecessary separate function as genesis calls both account::initialize and this. I merged this into account::initialize